src/onboarding/registration/RegulatoryTerms.test.tsx

```diff
@@ -4,26 +4,14 @@ import 'react-native'
 import { Provider } from 'react-redux'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import RegulatoryTerms, {
-  RegulatoryTerms as RegulatoryTermsClass,
-} from 'src/onboarding/registration/RegulatoryTerms'
+import { RegulatoryTerms as RegulatoryTermsClass } from 'src/onboarding/registration/RegulatoryTerms'
 import { createMockStore, getMockI18nProps } from 'test/utils'
 
 jest.mock('src/navigator/NavigationService', () => {
   return { navigate: jest.fn() }
 })
 
 describe('RegulatoryTermsScreen', () => {
-  it('renders correctly', () => {
```

Comment:
Should we update the snapshot instead of deleting this test to confirm expected changes? I know we mostly prefer not to use them, but in this case it seems valuable.

Reply:
I thought we agreed as a team to stop using them, also the only thing that was changing was the color of the button

Reply:
Also the button color change was intentional